rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
Does the nodejs package really come with npm on gentoo? npm is typically a layer on top of nodejs.